doctest/parts/doctest_fwd.h

```diff
@@ -1131,6 +1130,28 @@ DOCTEST_INTERFACE String toString(const Approx& in);
 
 DOCTEST_INTERFACE const ContextOptions* getContextOptions();
 
+#if DOCTEST_MSVC
+#define DOCTEST_INTERFACE_DECL
```

Comment:
Isn't this the same as what's near line 320? Seems that this is a redundant redefinition.

Reply:
Yep! Must've missed removing this when moving it to the top